application/F3DOptionsParser.cxx

```diff
@@ -374,11 +392,48 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     cxxOptions.positional_help("file1 file2 ...");
     cxxOptions.parse_positional({ "input" });
     cxxOptions.show_positional_help();
+    cxxOptions.allow_unrecognised_options();
 
     if (parseCommandLine)
     {
       auto result = cxxOptions.parse(this->Argc, this->Argv);
 
+      auto unmatched = result.unmatched();
+
+      if (unmatched.size() > 0)
+      {
+        constexpr int distanceThreshold = 5;
+
+        for (std::string unknownOption : unmatched)
+        {
+          // check if it's a long option
+          if (unknownOption[0] == '-' && unknownOption[1] == '-')
+          {
+            // remove trailing '--'
```

Comment:
should we check for long options using `-` instead of `--` ?

Reply:
nop, `-output` is not a typo, it's a mix of several short options